### PR TITLE
Updated Italian server in regional-hub

### PR DESCRIPTION
**Pull request by RyleeeeeUwU**
Fixed Italian server link to send to the actual server

### CODE REVIEW

File: wiki/communities/regional-hub.md [hunk 1]
```diff
@@ -109,11 +109,11 @@ German Beat Saber Community, JOIN US NOW! We have Magician! 😎
 
 ### Italy
 
-#### Italian Beat Saber Community
+#### BSITA
 
-A chill and accessible place for Italian players to hang around and meet each other
+A community for Italian Beat Saber players (Gio is here)
```
Comment (IAMDEDJUSTTOGETFILES) [inline wiki/communities/regional-hub.md:114]: Would rather have it say "The official server of Beat Saber Italia! Open for international players too!" to match the BSMG 
Discord

Comment (RyleeeeeUwU) [reply wiki/communities/regional-hub.md:114]: yeah sure

**Review by RyleeeeeUwU — COMMENTED** [review]
Updated the description as suggested

**Review by RyleeeeeUwU — COMMENTED** [review]
Updated description as suggested (oops I did it twice)

**Review by bloodcloak — APPROVED** [review]

**Pull request merged**